nion/swift/ExportDialog.py

```diff
@@ -1,7 +1,9 @@
 from __future__ import annotations
 
+
 # standard libraries
 import functools
+import enum
```

Comment:
Sorry it looks like it's still not alphabetical (my instructions were wrong). But `import functools` can be removed (for now) since it isn't used - and doing that will finally make this group of imports alphabetical.